src/main/java/com/ibanity/apis/client/products/ponto_connect/services/TokenService.java

```diff
@@ -0,0 +1,20 @@
+package com.ibanity.apis.client.products.ponto_connect.services;
+
+import com.ibanity.apis.client.products.ponto_connect.models.Token;
+
+import java.util.Map;
+
+public interface TokenService {
+
+    void revoke(String token, String clientSecret);
+
+    void revoke(Map<String, String> additionalHeaders, String token, String clientSecret);
+
+    Token create(String authorizationCode, String codeVerifier, String redirectUri, String clientSecret);
+
+    Token create(Map<String, String> additionalHeaders, String authorizationCode, String codeVerifier, String redirectUri, String clientSecret);
+
+    Token refresh(String refreshToken, String redirectUri, String clientSecret);
```

Comment:
why give the clientSecret in the refresh? It should be clientId + clientSecret or none if kept in the config, no?

Reply:
clientId and clientSecret are static but I didn't store the clientSecret as a field variable to avoid security issues (value read in the memory).